doc/project/working_groups.rst

```diff
@@ -0,0 +1,117 @@
+TSC Working Groups
+******************
+
+Overview
+########
+
+The TSC, at its discretion, may establish working groups or subcommittees to
+serve as focused teams dedicated to specific technical areas, initiatives, or
+tasks.
+
+Membership
+##########
+
+Working Group Membership Eligibility
+++++++++++++++++++++++++++++++++++++
+
+- Each Working group (WG) shall determine its own membership eligibility, in
+  consultation with the TSC.
+- Each working group shall have a core team of members who are actively involved
+  in its activities and decision-making processes.
+- It is expected that WG core membership shall be open to all Zephyr project
+  Contributors; however, working groups may impose restrictions such as the
+  number of participants from a single company.
+- The minimal number of core members may vary depending on the complexity of the
+  tasks and the breadth of expertise required to address them effectively.
+- A working group should aim to have at least five to seven core members to
+  ensure diversity of perspectives, collaboration, and continuity.
+- All TSC members are eligible to join a working group as core members, part of
```

Comment:
It is not fully clear if core members positions are restricted to project members or open to all Zephyr project contributors.

Reply:
In the third bullet point it says: "It is expected that WG core membership shall be open to all Zephyr project Contributors;". I think this part points out the TCS member role and responsibilities in the working group context

Reply:
but this will be duplication, no? :)
I just move the bullet up and emphasised "open to all Zephyr project Contributors"